config/initializers/assets.rb

```diff
@@ -1,39 +1,11 @@
-# Be sure to restart your server when you modify this file.
+# require 'extensions/propshaft/asset'
```

Comment:
Was this for debugging?

```suggestion
```

Reply:
oh that was copied from the idp and i forgot to remove it! thanks

Reply:
updated in https://github.com/18F/identity-dashboard/pull/740/commits/594bc2dafc173305e4c2b377dcf9ad97baabc8e9